itest/matcher/matcher.go

```diff
@@ -24,7 +24,7 @@ var (
 		Version: "0.28.1",
 	}
 	builtInScanner = v1alpha1.Scanner{
-		Name:    "Trivy-Operator",
+		Name:    "Trivy",
```

Comment:
should `"Trivy"` be a constant so we can re-use it around the code-base?

Reply:
> should `"Trivy"` be a constant so we can re-use it around the code-base?

Sure, it makes sense and easy to maintain

Reply:
Sounds good. The only potential issue is dependencies between packages. Any best-practice on where to put such constant in this codebase? I am not too familiar, at least not yet.

Reply:
Since this is a user-facing value, I suggest somewhere under the apis-tree. That should also be safe - dependency wise.

Reply:
I would add it to `config_audit_types.go`

Reply:
Added constant to `common_types.go`, PTAL!